uws/UWS/models.py

```diff
@@ -1,6 +1,34 @@
 # -*- coding: utf-8 -*-
 from lxml import etree as et
 
+uws_1_namespace = "http://www.ivoa.net/xml/UWS/v1.0"
+#uws_2_namespace = "http://www.ivoa.net/xml/UWS/v2.0"
+
+
+class UWS_1_Names(object): 
```

Comment:
Maybe (later) we can find a way to avoid passing in this class all the time in the init of the models. I will think about a solution (the one that currently comes to my mind is insane...)